### PR TITLE
feat: remote webcomponents

### PR DESCRIPTION
**Pull request by markuczy**

### CODE REVIEW

File: src/app/shell/services/routes.service.ts [hunk 1]
```diff
@@ -98,7 +103,15 @@ export class RoutesService implements ShowContentProvider {
           ? r.exposedModule.slice(2)
           : r.exposedModule;
         console.log(`Load remote module ${exposedModule} finished.`);
-        return m[exposedModule];
+        if (
+          r.technology === Technologies.Angular &&
+          r.appId !== 'hello-webcomponent-ui' &&
+          r.appId !== 'onecx-announcement-ui'
+        ) {
```
Comment (markuczy) [inline src/app/shell/services/routes.service.ts:110]: tmp workaround

File: src/app/shell/services/routes.service.ts [hunk 2]
```diff
@@ -77,7 +82,7 @@ export class RoutesService implements ShowContentProvider {
       path: this.toRouteUrl(r.baseUrl),
       data: {
         module: r.exposedModule,
-        breadcrumb: r.productName,
+        mfeInfo: this.routeToMfeInfo(r, r.baseUrl),
```
Comment (markuczy) [inline src/app/shell/services/routes.service.ts:85]: change will be required in all uis once shell is udpated

File: src/app/app.component.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { Component } from '@angular/core';
+import { Component, NgZone, inject } from '@angular/core';
```
Comment (SchettlerKoehler) [inline src/app/app.component.ts:1]: Why is this needed?

**Review by SchettlerKoehler — CHANGES_REQUESTED** [review]

File: src/app/app.component.ts (hunk 3 above)
Comment (markuczy) [reply src/app/app.component.ts:1]: its not needed anymore

**Pull request merged**